security/auth_providers.rst

```diff
@@ -1,23 +1,88 @@
-.. index::
-   single: Security; Pre authenticated providers
+Built-in Authentication Providers
+=================================
 
-Using pre Authenticated Security Firewalls
-==========================================
+If you need to add authentication to your app, we recommend using
```

Comment:
We avoid the use of "we" in all docs ... but I'm not 100% convinced by that rule. I don't think that using "we" is wrong ... jus pointing that right now it would be inconsistent.

Reply:
I'm aware of the existing rule... and I've been gently ignoring it for a while now. I realize that's not ideal... we should find a rule (even if it means allowing `we`) and be consistent.

I like we better here because the alternative uses a passive voice "it is recommended..."